test/e2e/federated-service.go

```diff
@@ -62,11 +62,22 @@ var FederatedServiceLabels = map[string]string{
 	"foo": "bar",
 }
 
+/*
+type cluster keeps track of the assorted objects and state related to each
```

Comment:
Convention is to just start with the entity's name - "cluster keeps track of ..."